test_scripts/RC/CLIMATE_RADIO/OnRemoteControlSettings/025_IN_USE_in_case_3_requests.lua

```diff
@@ -0,0 +1,224 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_requirements/issues/10
+-- Use case: https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/resource_allocation.md
+-- Item: Use Case 1: Exception 2
+--
+-- User story: https://github.com/smartdevicelink/sdl_requirements/issues/9
+-- Use case: https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/button_press_emulation.md
+-- Item: Use Case 1: Exception 1.5
+--
+-- User story: https://github.com/smartdevicelink/sdl_requirements/issues/3
+-- Use case: https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/SetInteriorVehicleData.md
+-- Item: Use Case 1: Exception 6.1
+--
+-- Requirement summary:
+-- [SDL_RC] Resource allocation based on access mode
+--
+-- Description: TRS: OnRemoteControlSettings, #3
+-- In case:
+-- 1) SDL received OnRemoteControlSettings notification from HMI with allowed:true
+-- 2) and "accessMode" = "AUTO_ALLOW" or without "accessMode" parameter at all
+-- 3) and RC_module on HMI is alreay in control by RC-application_1
+-- 4) and RC_module is currently executing request by RC_application_1
+-- 5) and another RC_application_2 in HMILevel LIMITED sends control RPC (either SetInteriorVehicleData or ButtonPress)
+-- 6) and another RC_application_3 in HMILevel FULL sends control RPC (either SetInteriorVehicleData or ButtonPress)
```

Comment:
Why would `RC_application_3` be rejected in this case? Shouldn't any app that requests a module in `FULL` under `AUTO_ALLOW` be allowed?

Reply:
SDL must` 2) not process the request from RC_application_2, RC_application_3 and respond with result code IN_USE, success:false` because `RC_module is currently executing request by RC_application_1`

Reply:
I see, I missed the detail that the request from App 1 was still in progress